content/en/providers/operations-manuals/howto20_registering_service_gocdb_required_fields/_index.md

```diff
@@ -0,0 +1,51 @@
+---
+title: HOWTO20
+weight: 135
+type: "docs"
+description: "Registering a new service in GOC-DB: mandatory and optional information"
+---
+
+Information to provide when registering a new service (the fields marked
+with an asterisk are mandatory):
+
+`Hosting Site * (select the appropriate RC under which you are registering the service)`
```

Comment:
would this look better on a table? or maybe a list?

Reply:
Added as a list for now